lua/entities/ent_jack_turret_base/init.lua

```diff
@@ -690,15 +725,28 @@ function ENT:FireShot()
         end
     end
 
+    local SpreadVec = Vector() -- manually build spread to avoid weird bug where all shots land in the top right for no apparent reason
+
+    SpreadVec.x = spread * math.Rand( -1, 1 )
+    SpreadVec.y = spread * math.Rand( -1, 1 )

```

Comment:
```suggestion
    local SpreadVec = Vector( spread, spread, spread ) + VectorRand( -1, 1 )
```
you can multiply the `VectorRand` by some `intensity` amount I think

Reply:
Applied, now uses VectorRand as the "random creator".
Z is behind/in front, just makes the turrets biast towards shooting in the center.